third_party/move/move-compiler-v2/src/pipeline/reference_safety_processor.rs

```diff
@@ -1793,7 +1793,9 @@ impl<'env, 'state> LifetimeAnalysisStep<'env, 'state> {
                 {
                     continue;
                 }
-                if other_label == label {
+                if other_label == label
```

Comment:
apart from the same memory location, locations mutably borrowed from `label` are also included.

Reply:
Could we add this as a comment here, please?

Reply:
@rahxephon89 Just double checking if this comment was addressed.